core/domain/exp_services.py

```diff
@@ -786,7 +786,10 @@ def delete_explorations(committer_id, exploration_ids, force_deletion=False):
 
 
 def delete_explorations_from_subscribed_users(exploration_ids):
-    """Remove explorations from all subscribers' entity_ids.
+    """Remove explorations from all subscribers' activity_ids.
+
+    Note: activity_ids in UserSubscriptionsModel are ought be renamed to
```

Comment:
Make this a TODO instead and point it to an issue. That way, whoever's fixing the issue can see this.

Reply:
Done.